db/deploy/removed-hotel.sql

```diff
@@ -0,0 +1,252 @@
+-- Deploy rams-core:removed-hotel to pg
```

Comment:
Hm... in _theory_ you should be able to run migrations on core, then go into each plugin and run its migrations. Of course that means each plugin needs its own /db/ folder, and the raw files you make will only be correct if you run them on a DB that has only that plugin and core and nothing else. Which is a bit cumbersome, to be sure.

Per-project migrations sound fine. Obviously though the migrations in core should be only for core stuff.